CHANGELOG.md

```diff
@@ -5,11 +5,12 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
-- Allow configuration of static and dynamic node priorities in compute resources via the ParallelCluster configuration YAML file.
+- Allow configuration of static and dynamic node priorities in Slurm compute resources via the ParallelCluster configuration YAML file.
 
 **CHANGES**
 
 **BUG FIXES**
+- Fix the bad prioritization of Slurm idle dynamic nodes over static ones due to alphabetical sorting of node names.
```

Comment:
What about 
```
Fix issue causing dynamic nodes to be occasionally prioritized over static nodes.
```

Or if we want to be more explicit on the root cause replacing "occasionally" with "due to alphabetical sorting of node names"